packages/sources/src/styling/global/dl.scss

```diff
@@ -89,44 +89,6 @@ dl {
     > div {
       margin-bottom: $u2;
     }
-
-    @media screen and (min-width: $screen-sm-min) {
```

Comment:
Ik begrijp niet waarom deze code weg kan, hoe zit dit? Percy klaagt niet, is deze styling dubbel?

Reply:
Dit is inderdaad dubbel, dezelfde stijling begint op regel 51 in dit bestand